assets/backstop_data/engine_scripts/casper/loadCookies.js

```diff
@@ -0,0 +1,15 @@
+var fs = require('fs');
+
+module.exports = function (casper, scenario) {
+  var cookies = [];
```

Comment:
I'm not quite sure that this variable require to be initialized with an empty array

Reply:
Oh all of these are not mine 😸